tgui/packages/tgui/interfaces/Pressurizer.js

```diff
@@ -0,0 +1,223 @@
+  };
+
+  return (
+    <Window
+      theme={emagged ? 'syndicate' : 'ntos'}
+      width={390}
+      height={390}>
+      <Window.Content>
+        <PortableBasicInfo
+          connected={connected}
+          pressure={pressure}
+          maxPressure={maxPressure}>
+          <Divider />
+          <LabeledList>
+            <LabeledList.Item label="Emergency Blast Release">
+              <Button
+                fluid
+                textAlign="center"
+                icon="circle"
+                content={getArmedState()}
+                disabled={sufficientPressure() || !airSafe}
+                color={blastArmed ? 'bad' : 'average'}
+                onClick={() => handleArmPressurizer()} />
+            </LabeledList.Item>
+            <LabeledList.Item label="Delay">
+              <Button onClick={() => handleSetBlastDelay(minArmDelay)}>
+                Min
+              </Button>
+              <NumberInput
+                animated
+                width="7em"
+                value={blastDelay}
+                minValue={minArmDelay}
+                maxValue={maxArmDelay}
+                onChange={(e, targetDelay) => handleSetBlastDelay(targetDelay)} />
+              <Button onClick={() => handleSetBlastDelay(maxArmDelay)}>
+                Max
+              </Button>
+            </LabeledList.Item>
+          </LabeledList>
+          <Divider />
+          <LabeledList>
+            <LabeledList.Item label="Fan Status">
+              <Button
+                color={fanState === FanState.Off ? 'bad' : 'default'}
+                onClick={() => handleSetFan(FanState.Off)}
+              >
+                Off
+              </Button>
+              <Button
+                color={fanState === FanState.In ? 'good' : 'default'}
+                onClick={() => handleSetFan(FanState.In)}
+              >
+                In
+              </Button>
+              <Button
+                color={fanState === FanState.Out ? 'good' : 'default'}
+                onClick={() => handleSetFan(FanState.Out)}
+              >
+                Out
+              </Button>
+            </LabeledList.Item>
+            <LabeledList.Item label="Release Pressure">
+              <Button onClick={() => handleSetPressure(minRelease)}>
+                Min
+              </Button>
+              <NumberInput
+                animated
+                width="7em"
+                value={releasePressure}
+                minValue={minRelease}
+                maxValue={maxRelease}
+                onChange={(e, targetPressure) => handleSetPressure(targetPressure)} />
+              <Button onClick={() => handleSetPressure(maxRelease)}>
+                Max
+              </Button>
+            </LabeledList.Item>
+          </LabeledList>
+        </PortableBasicInfo>
+        <Section
+          title="Material Processing"
+          buttons={(
+            <Button
+              icon="eject"
+              disabled={!materialsCount}
```

Comment:
```suggestion
              disabled={materialsCount === 0}
```
I know `!materialsCount` would work here, but better to convey that this is a number rather than a boolean.